packages/server/graphql/mutations/helpers/createNewMeetingPhases.ts

```diff
@@ -132,6 +133,20 @@ const createNewMeetingPhases = async (
         case LAST_CALL:
         case 'SCOPE':
           return new GenericMeetingPhase(phaseType, durations)
+        case 'RESPONSES':
+          const teamMembers = await dataLoader.get('teamMembersByTeamId').load(teamId)
+          const teamMemberIds = teamMembers.map(({id}) => id)
+          const teamPromptResponsesPhase = new TeamPromptResponsesPhase(teamMemberIds)
+          const {stages: teamPromptStages} = teamPromptResponsesPhase
+          const teamMemberResponseDiscussion = teamPromptStages.map((stage) => ({
+            id: stage.discussionId,
+            teamId,
+            meetingId,
+            discussionTopicId: stage.teamMemberId,
```

Comment:
-1 This will cause different team prompt meetings have the same `discussionTopicId`, yes?

Reply:
this may sound dumb but does `discussionTopicId` have to be globally unique? my understanding is it has to be unique within the meeting but I'm open to changing that if I overlooked something!